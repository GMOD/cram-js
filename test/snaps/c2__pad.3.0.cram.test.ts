import { testFile } from '../testUtil'
testFile('c2#pad.3.0.cram')
