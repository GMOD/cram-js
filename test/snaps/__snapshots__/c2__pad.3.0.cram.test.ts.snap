// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`can dump the whole c2#pad.3.0.cram without error 1`] = `
[
  {
    "fileId": "-",
    "magic": "CRAM",
    "majorVersion": 3,
    "minorVersion": 0,
  },
  {
    "containerHeader": {
      "_endPosition": 45,
      "_size": 19,
      "alignmentSpan": 0,
      "crc32": 4062659929,
      "landmarks": [
        0,
        300,
      ],
      "length": 461,
      "numBases": 0,
      "numBlocks": 2,
      "numLandmarks": 2,
      "numRecords": 0,
      "recordCounter": 0,
      "refSeqId": 0,
      "refSeqStart": 0,
    },
    "data": [
      {
        "_endPosition": 45,
        "_size": 19,
        "alignmentSpan": 0,
        "crc32": 4062659929,
        "landmarks": [
          0,
          300,
        ],
        "length": 461,
        "numBases": 0,
        "numBlocks": 2,
        "numLandmarks": 2,
        "numRecords": 0,
        "recordCounter": 0,
        "refSeqId": 0,
        "refSeqStart": 0,
      },
      {
        "samHeader": [
          {
            "data": [
              {
                "tag": "SN",
                "value": "c2",
              },
              {
                "tag": "LN",
                "value": "9",
              },
              {
                "tag": "M5",
                "value": "ccad919bff6a8182a0ccf0714a2ca4c9",
              },
              {
                "tag": "UR",
                "value": "/home/vadim/bin/htslib/test/c2.fa",
              },
            ],
            "tag": "SQ",
          },
          {
            "data": [
              {
                "tag": "mpileup example from https",
                "value": "//github.com/samtools/htslib/issues/59",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "with additional Pad cigar operations",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": " c2    CC***AA**T**AA***CC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s1    CT***AA**T**AA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s1b   CT*******T*******TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s2    CT*****G***G*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s2p   CT*****G***G*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s3    CT*****GG*GG*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s3b   CT****CGGCGGC****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s4    CT***AAG***GAA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s4p   CT***AAG***GAA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s5    CTGGG*********GGGTC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
        ],
      },
      {
        "samHeader": [
          {
            "data": [
              {
                "tag": "SN",
                "value": "c2",
              },
              {
                "tag": "LN",
                "value": "9",
              },
              {
                "tag": "M5",
                "value": "ccad919bff6a8182a0ccf0714a2ca4c9",
              },
              {
                "tag": "UR",
                "value": "/home/vadim/bin/htslib/test/c2.fa",
              },
            ],
            "tag": "SQ",
          },
          {
            "data": [
              {
                "tag": "mpileup example from https",
                "value": "//github.com/samtools/htslib/issues/59",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "with additional Pad cigar operations",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": " c2    CC***AA**T**AA***CC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s1    CT***AA**T**AA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s1b   CT*******T*******TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s2    CT*****G***G*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s2p   CT*****G***G*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s3    CT*****GG*GG*****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s3b   CT****CGGCGGC****TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s4    CT***AAG***GAA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s4p   CT***AAG***GAA***TC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
          {
            "data": [
              {
                "tag": "+s5    CTGGG*********GGGTC",
                "value": "",
              },
            ],
            "tag": "CO",
          },
        ],
      },
    ],
  },
  {
    "compressionScheme": {
      "APdelta": true,
      "dataSeriesEncoding": {
        "AP": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "BA": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              71,
            ],
          },
          "parametersBytes": 4,
        },
        "BB": {
          "codecId": 4,
          "parameters": {
            "lengthsEncoding": {
              "codecId": 1,
              "parameters": {
                "blockContentId": 42,
              },
              "parametersBytes": 1,
            },
            "valuesEncoding": {
              "codecId": 1,
              "parameters": {
                "blockContentId": 37,
              },
              "parametersBytes": 1,
            },
          },
          "parametersBytes": 6,
        },
        "BF": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "BS": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              1,
              2,
              2,
            ],
            "numCodes": 3,
            "numLengths": 3,
            "symbols": [
              2,
              0,
              1,
            ],
          },
          "parametersBytes": 8,
        },
        "CF": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              3,
            ],
          },
          "parametersBytes": 4,
        },
        "DL": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              1,
              2,
              2,
            ],
            "numCodes": 3,
            "numLengths": 3,
            "symbols": [
              2,
              1,
              5,
            ],
          },
          "parametersBytes": 8,
        },
        "FC": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              1,
              2,
              3,
              4,
              4,
            ],
            "numCodes": 5,
            "numLengths": 5,
            "symbols": [
              88,
              68,
              105,
              73,
              80,
            ],
          },
          "parametersBytes": 12,
        },
        "FN": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              2,
              2,
              2,
              3,
              3,
            ],
            "numCodes": 5,
            "numLengths": 5,
            "symbols": [
              5,
              7,
              9,
              2,
              4,
            ],
          },
          "parametersBytes": 12,
        },
        "FP": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              2,
              2,
              2,
              3,
              3,
            ],
            "numCodes": 5,
            "numLengths": 5,
            "symbols": [
              0,
              1,
              2,
              3,
              6,
            ],
          },
          "parametersBytes": 12,
        },
        "IN": {
          "codecId": 5,
          "parameters": {
            "blockContentId": 13,
            "stopByte": 0,
          },
          "parametersBytes": 2,
        },
        "MF": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "MQ": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "NP": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "NS": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              -1,
            ],
          },
          "parametersBytes": 8,
        },
        "PD": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              1,
            ],
          },
          "parametersBytes": 4,
        },
        "QS": {
          "codecId": 1,
          "parameters": {
            "blockContentId": 12,
          },
          "parametersBytes": 1,
        },
        "RG": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              -1,
            ],
          },
          "parametersBytes": 8,
        },
        "RI": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "RL": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              2,
              2,
              3,
              3,
              3,
              3,
            ],
            "numCodes": 6,
            "numLengths": 6,
            "symbols": [
              6,
              10,
              5,
              8,
              9,
              11,
            ],
          },
          "parametersBytes": 14,
        },
        "RN": {
          "codecId": 5,
          "parameters": {
            "blockContentId": 11,
            "stopByte": 0,
          },
          "parametersBytes": 2,
        },
        "SC": {
          "codecId": 5,
          "parameters": {
            "blockContentId": 14,
            "stopByte": 0,
          },
          "parametersBytes": 2,
        },
        "TL": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
        "TS": {
          "codecId": 3,
          "parameters": {
            "bitLengths": [
              0,
            ],
            "numCodes": 1,
            "numLengths": 1,
            "symbols": [
              0,
            ],
          },
          "parametersBytes": 4,
        },
      },
      "readNamesIncluded": true,
      "referenceRequired": false,
      "substitutionMatrix": [
        [
          "C",
          "G",
          "T",
          "N",
        ],
        [
          "A",
          "G",
          "T",
          "N",
        ],
        [
          "A",
          "C",
          "T",
          "N",
        ],
        [
          "A",
          "C",
          "G",
          "N",
        ],
        [
          "A",
          "C",
          "G",
          "T",
        ],
      ],
      "tagEncoding": {},
      "tagIdsDictionary": [
        [],
      ],
    },
    "containerHeader": {
      "_endPosition": 524,
      "_size": 18,
      "alignmentSpan": 9,
      "crc32": 482541517,
      "landmarks": [
        270,
      ],
      "length": 508,
      "numBases": 75,
      "numBlocks": 6,
      "numLandmarks": 1,
      "numRecords": 9,
      "recordCounter": 0,
      "refSeqId": 0,
      "refSeqStart": 1,
    },
    "data": [
      {
        "_endPosition": 524,
        "_size": 18,
        "alignmentSpan": 9,
        "crc32": 482541517,
        "landmarks": [
          270,
        ],
        "length": 508,
        "numBases": 75,
        "numBlocks": 6,
        "numLandmarks": 1,
        "numRecords": 9,
        "recordCounter": 0,
        "refSeqId": 0,
        "refSeqStart": 1,
      },
      {
        "features": [
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              55,
              55,
              55,
              55,
              55,
              55,
              55,
              55,
              55,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 8,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 9,
            "readName": "s1",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 800,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 4,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 4,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 5,
            "readName": "s1b",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 801,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 3,
                "refPos": 5,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 4,
                "refPos": 5,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 4,
                "refPos": 6,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 5,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 5,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 6,
            "readName": "s2",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 802,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 3,
                "refPos": 5,
              },
              {
                "code": "P",
                "data": 1,
                "pos": 4,
                "refPos": 5,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 4,
                "refPos": 5,
              },
              {
                "code": "P",
                "data": 1,
                "pos": 4,
                "refPos": 6,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 4,
                "refPos": 6,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 5,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 5,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 6,
            "readName": "s2p",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 803,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "I",
                "data": "GG",
                "pos": 3,
                "refPos": 5,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 5,
                "refPos": 5,
              },
              {
                "code": "I",
                "data": "GG",
                "pos": 5,
                "refPos": 6,
              },
              {
                "code": "D",
                "data": 2,
                "pos": 7,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 7,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 8,
            "readName": "s3",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 804,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "X",
                "data": 0,
                "pos": 3,
                "refPos": 4,
              },
              {
                "code": "I",
                "data": "GG",
                "pos": 4,
                "refPos": 5,
              },
              {
                "code": "X",
                "data": 1,
                "pos": 6,
                "refPos": 5,
              },
              {
                "code": "I",
                "data": "GG",
                "pos": 7,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 0,
                "pos": 9,
                "refPos": 6,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 10,
                "refPos": 7,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 10,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 11,
            "readName": "s3b",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 805,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 5,
                "refPos": 5,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 6,
                "refPos": 5,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 6,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 9,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 10,
            "readName": "s4",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 806,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 5,
                "refPos": 5,
              },
              {
                "code": "P",
                "data": 1,
                "pos": 6,
                "refPos": 5,
              },
              {
                "code": "D",
                "data": 1,
                "pos": 6,
                "refPos": 5,
              },
              {
                "code": "P",
                "data": 1,
                "pos": 6,
                "refPos": 6,
              },
              {
                "code": "i",
                "data": "G",
                "pos": 6,
                "refPos": 6,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 9,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 10,
            "readName": "s4p",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 807,
          },
          {
            "alignmentStart": 1,
            "cramFlags": 3,
            "flags": 0,
            "lengthOnRef": 9,
            "mappingQuality": 0,
            "qualityScores": [
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
              255,
            ],
            "readBases": undefined,
            "readFeatures": [
              {
                "code": "X",
                "data": 2,
                "pos": 2,
                "refPos": 2,
              },
              {
                "code": "I",
                "data": "GGG",
                "pos": 3,
                "refPos": 3,
              },
              {
                "code": "D",
                "data": 5,
                "pos": 6,
                "refPos": 3,
              },
              {
                "code": "I",
                "data": "GGG",
                "pos": 6,
                "refPos": 8,
              },
              {
                "code": "X",
                "data": 2,
                "pos": 9,
                "refPos": 8,
              },
            ],
            "readGroupId": -1,
            "readLength": 10,
            "readName": "s5",
            "sequenceId": 0,
            "tags": {},
            "templateSize": 0,
            "uniqueId": 808,
          },
        ],
        "header": {
          "_endPosition": 834,
          "_size": 35,
          "compressedSize": 31,
          "compressionMethod": "raw",
          "content": Uint8Array [
            0,
            1,
            9,
            9,
            0,
            4,
            3,
            11,
            12,
            13,
            255,
            255,
            255,
            255,
            15,
            204,
            173,
            145,
            155,
            255,
            106,
            129,
            130,
            160,
            204,
            240,
            113,
            74,
            44,
            164,
            201,
          ],
          "contentId": 0,
          "contentPosition": 799,
          "contentType": "MAPPED_SLICE_HEADER",
          "crc32": 2201063245,
          "parsedContent": {
            "_endPosition": 555,
            "_size": 31,
            "contentIds": [
              11,
              12,
              13,
            ],
            "md5": [
              204,
              173,
              145,
              155,
              255,
              106,
              129,
              130,
              160,
              204,
              240,
              113,
              74,
              44,
              164,
              201,
            ],
            "numBlocks": 4,
            "numContentIds": 3,
            "numRecords": 9,
            "recordCounter": 0,
            "refBaseBlockId": -1,
            "refSeqId": 0,
            "refSeqSpan": 9,
            "refSeqStart": 1,
          },
          "uncompressedSize": 31,
        },
      },
    ],
  },
]
`;
